apps/web/src/components/ProjectCard/ProjectCard.tsx

```diff
@@ -77,53 +65,58 @@ const ProjectCard = ({ project }: ProjectCardProps) => {
   };
 
   return (
-    <div className={projectCardStyle}>
+    <div className='w-[330px]'>
       <LoginModal
         isOpen={isLoginModalOpen}
         onClose={() => setIsLoginModalOpen(false)}
       />
-      <Link
-        href={{
-          pathname: `/project/${project.id}`,
-        }}
-        passHref
-      >
-        <a className={imageContainerStyle}>
+      <div className='flex cursor-pointer'>
+        <Link
+          href={{
+            pathname: `/project/${project.id}`,
+          }}
+          passHref
+        >
           <Image
-            className={previewStyle}
+            className='rounded-t-lg object-cover transition ease-in-out duration-300 hover:brightness-75 hover:opacity-100  '
```

Comment:
could you remove this extra space at the end?

Reply:
sure